src/hypothesis/internal/floats.py

```diff
@@ -48,12 +48,31 @@ def count_between_floats(x, y):
 
 
 def float_to_int(value):
-    return (
-        struct_unpack(b'!Q', struct_pack(b'!d', value))[0]
-    )
+    return struct_unpack(b'!Q', struct_pack(b'!d', value))[0]
 
 
 def int_to_float(value):
-    return (
-        struct_unpack(b'!d', struct_pack(b'!Q', value))[0]
-    )
+    return struct_unpack(b'!d', struct_pack(b'!Q', value))[0]
+
+
+def next_up(value):
```

Comment:
I read through the linked SO post. Annoying that this isn't in the standard library!